README.md

```diff
@@ -37,6 +37,7 @@ config := bigcache.Config{
 		MaxEntriesInWindow: 1000 * 10 * 60, // rps * lifeWindow
 		MaxEntrySize: 500,                  // max entry size in bytes, used only in initial memory allocation
 		Verbose: true,                      // prints information about additional memory allocation
+		HardMaxCacheSize: 8192,             // cache will not allocate more memory than this limit, value in MB, 0 means no limit
```

Comment:
It is described in config.go but ok I have added also info here. As for `Hard` prefix I decided to leave it because we already have `MaxEntriesInWindow` and `MaxEntrySize` which are soft limits - used only in initial memory allocation, therefore I wanted to distinct them.